packages/components/src/responsive-toolbar-group/dropdown-group.tsx

```diff
@@ -82,45 +90,49 @@ export default function DropdownGroup( {
 
 		if ( containGroupedIndexes || always ) {
 			return (
-				<Dropdown
-					renderToggle={ ( { onToggle } ) => (
-						<ToolbarButton
-							className={ classnames(
-								'responsive-toolbar-group__more-item',
-								'responsive-toolbar-group__button-item'
-							) }
-							isActive={ groupedIndexes[ activeIndex ] }
-							onClick={ () => {
-								onToggle();
-							} }
-						>
-							{ translate( 'More' ) }
-							<Icon icon={ chevronDown } />
-						</ToolbarButton>
-					) }
-					renderContent={ ( { onClose } ) => (
-						<MenuGroup>
-							{ getChildrenToRender()
-								.filter( ( { grouped } ) => grouped )
-								.map( ( { index, child } ) => (
-									<MenuItem
-										key={ `menu-item-${ index }` }
-										onClick={ () => {
-											setActiveIndex( parseInt( index ) );
-											onClick( parseInt( index ) );
-											onClose();
-										} }
-										className={ classnames(
-											'responsive-toolbar-group__menu-item',
-											activeIndex === parseInt( index ) ? 'is-selected' : ''
-										) }
-									>
-										{ child }
-									</MenuItem>
-								) ) }
-						</MenuGroup>
-					) }
-				/>
+				<SlotFillProvider>
+					{ /* @ts-expect-error-ignore  */ }
+					<Popover.Slot />
+					<Dropdown
```

Comment:
It's the same component. I just moved the Dropdown component inside of the SlotFillProvider component.